### PR TITLE
Turkish language added to translations

### PR DESCRIPTION
**Pull request by ubeydeozdmr**

### CODE REVIEW

File: countries.json [hunk 1]
```diff
@@ -32209,6 +33037,10 @@
                 "official": "Demokratiska republiken S\u00e3o Tom\u00e9 och Pr\u00edncipe",
                 "common": "S\u00e3o Tom\u00e9 och Pr\u00edncipe"
             },
+            "tur": {
+				"official": "São Tomé ve Príncipe Demokratik Cumhuriyeti",
```
Comment (mledoze) [inline countries.json:33041]: Could you JSON encode accented characters? Not just here but for all turkish translations

Comment (ubeydeozdmr) [reply countries.json:33041]: Now I'm done doing it

**Comment by ubeydeozdmr** [comment]
Is there anything else I need to edit?

**Comment by mledoze** [comment]
Thank you for your contribution! It's merged.

**Pull request merged**